tsconfig.json

```diff
@@ -40,5 +40,5 @@
 	"include": [
```

Comment:
getting an error from this file ': Unexpected token } in JSON at position 1069' during tests